tests/calc/test_thermo.py

```diff
@@ -31,6 +31,7 @@
                         vertical_velocity, vertical_velocity_pressure,
                         virtual_potential_temperature, virtual_temperature,
                         wet_bulb_temperature)
+from metpy.calc.exceptions import InvalidSoundingError
```

Comment:
Given that you needed it here for the test and other users *might* want to catch it, can you instead change this to import it from `metpy.calc`, and add `from .exceptions import InvalidSoundingError` to `src/metpy/calc/__init__.py`?

Reply:
Makes sense!